rest-api-spec/src/main/resources/rest-api-spec/test/index/110_strict_allow_templates.yml

```diff
@@ -0,0 +1,155 @@
+---
+"Index documents with setting dynamic parameter to strict_allow_templates in the mapping of the index":
+  - skip:
+      version: " - 2.99.99"
```

Comment:
Needs update to `2.15.99`, will do this shortly

Reply:
Yes, this enhancement targets for 2.16.0, it should be backported to 2.x.

Reply:
Thanks @gaobinlong , I will alter this pull request first thing in the morning